word2vec.py

```diff
@@ -0,0 +1,138 @@
+from gensim.models import Word2Vec, KeyedVectors
+from sklearn.metrics import roc_curve, roc_auc_score, auc
+import pandas as pd
+import numpy as np
+import random
+import os
+import matplotlib.pyplot as plt
+
+
+class Sentences(object):
+    """
+    Extracts title + abstracts from Pubtator data. Replaces any instance of a 
+    gene or disease with the Entrez gene ID or MESH ID, respectively. 
+    """
+    def __init__(self, filename):
+        self.filename = filename
+ 
+    def __iter__(self):
+        curr_pmid = curr_title = curr_abstract = curr_total = None
+        for line in open(self.filename):
```

Comment:
```suggestion
        for line in open(self.filename, 'r'):
```
It is good to be explicit on the file mode just so you don't accidentally open a file in writing mode and undo everything.